index.html

```diff
@@ -2530,6 +2530,78 @@ <h5>Additional Properties in the Manifest</h5>
+							<li><code>undefined</code> otherwise</li>
+						</ul>
+					</li>
+					</li>
+			</ul>
+
+			<p>The steps of the algorithm are described below. As an abuse of notation, <code>P["term"]</code> refers to the value in the object <code>P</code> for the label <code>"term"</code>, where <code>P</code> is either <code>manifest</code>, or an object appearing within <code>manifest</code> (e.g., a <code>Person</code>).</p>
+			<ol>
+				<li>(<a href="#wptitle"></a>) if <code>manifest["name"]</code> is <code>undefined</code>, then locate the <a href="https://www.w3.org/TR/html/document-metadata.html#the-title-element"><code>title</code></a> HTML element&nbsp;[[!html]] using <code>document</code>. If that element exists, let <code>t</code> be its text content, and add to <code>manifest</code>:
+					<ul>
+						<li>if the language of <code>title</code> is explicitly <a href="https://www.w3.org/TR/html/dom.html#the-lang-and-xmllang-attributes">set</a> to the value of <code>l</code>, then add <br/><code>"name" : {"@value" : t, "@language" : l}</code><br/></li>
+						<li>or <br/><code>"name" : t</code><br> otherwise</li>
+					</ul>
+				</li>
+				<li>(<a href="#language-and-dir"></a>) if <code>manifest["inLanguage"]</code> is <code>undefined</code> and the value of <code>lang</code> is <em>not</em> <code>undefined</code>, add<br><code>"inLanguage" : lang</code><br> to <code>manifest</code> </li>
+				<li>(<a href="#language-and-dir"></a>) if <code>manifest["inDirection"]</code> is <code>undefined</code> and the value of <code>dir</code> is <em>not</em> <code>undefined</code>, add<br><code>"inDirection" : dir</code><br> to <code>manifest</code> </li>
+				<li>(<a href="#arrays-and-single-values"></a>) if the value of <code>P["term"]</code>, where <code>P</code> is any object in <code>manifest</code> (including itself) and <code>term</code> is:
+					<ul>
+						<li><code>@type</code>; or</li>
+						<li>one of the <a href="#accessibility">accessibility</a> terms except for <code>accessibilitySummary</code>; or</li>
+						<li>one of the <a href="#creators">creator</a> terms; or</li>
+						<li><code>name</code>; or</li>
+						<li>one of the <a href="#resource-categorization-properties">resource categorization properties</a>; or</li>
+						<li><code>rel</code></li>
+					</ul>
+					is a single string or object <code>v</code>, then change the relevant term/value to<br><code>"term" : [v]</code>
+				</li>
+				<li>(<a href="#creators"></a>) if one of values <code>v</code> in the <code>manifest["term"]</code> array, where <code>term</code> is one of the <a href="#creators">creator</a> terms, is a single string, exchange that element of the array to<br><code>"term" : { "@type": "Person", "name" : [v]}</code></li>
```

Comment:
Well... neither:-) Because this is step 4, everything is already an array, and this step refers to _one of the elements_ of the array. In other words, it should rather be simply `{"@type":"Person", "name":[v]}` without the `"term":`

Will update this in a few.

Reply:
Correct, that's step 4.

What's the point of step 5 then? Is this simply to add a default `"type": "Person"`? In this case, this should be applied when we have `@value` and `@language` as well (instead of a string).

But my other comment about Person/Organization in the WebIDL, applies here as well.

Reply:
No. It is to convert

```json
"author": ["John Doe"]
```

to 

```json
"author": [{"@type": "Person", "name": ["John Doe"]}]
```

(The text conversion to objects are done in a later step.)

Reply:
Right, that's what I meant as well.

But what about:
```
"author": [
  {
    "@value": "John Doe",
    "@language": "en"
  }
]
```

shouldn't this be converted to:

```
"author": [
  {
    "@type": "Person",
    "name": {
      "@value": "John Doe",
      "@language": "en"
    }
  }
]
```

Reply:
You are right, it should. I will have to rephrase it to say "string or object", or something like that. Will do.